acceptancetests/assess_model_migration_versions.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+"""Test migrating models between controllers of increasing versions."""
```

Comment:
Is there a specific reason to not expand the existing model migration test? I see instead you just import from assess_model_migration.